src/shared/FlashRound/FlashRoundAdmin.tsx

```diff
@@ -164,10 +164,19 @@ const FlashRoundAdmin = () => {
 
   const stopSound = (audio: any) => {
     const { stop } = useSounds(audio);
-    setPlayingCurrentThemeSong(false);
-    stop(audio);
+    for (let i = 0; i < 10; i++) {
```

Comment:
cool implementation for the fade out effect, in the useSound hook add a method that we export called fadeOut which will take audio as argument, export it just like play and stop and import it destructured at the top of the component